lxd/storage/drivers/driver_cephobject_utils.go

```diff
@@ -252,6 +253,13 @@ func (d *cephobject) radosgwadminBucketList(ctx context.Context) ([]string, erro
 }
 
 // radosgwBucketName returns the bucket name to use for the actual radosgw bucket.
-func (d *cephobject) radosgwBucketName(bucketName string) string {
+func (d *cephobject) radosgwBucketName(storageBucketName string) string {
+	bucketName := storageBucketName
+
+	// Remove project prefix if applicable.
+	if strings.Contains(bucketName, "_") {
+		_, bucketName = project.StorageVolumeParts(bucketName)
```

Comment:
We should just use:

```
_, bucketName := project.StorageVolumeParts(storageBucketName)
``` 

Otherwise it implies that having the project in the storageBucketName is optional.